spec/controllers/api/v1/orders_controller_spec.rb

```diff
@@ -529,9 +524,9 @@ def create_order_with_transport(state)
       end
 
       context 'Processing checklist' do
-        let!(:checklist_it1) { create :process_checklist, booking_type: booking_type }
-        let!(:checklist_it2) { create :process_checklist, booking_type: booking_type }
-        let!(:checklist_it3) { create :process_checklist, booking_type: booking_type }
+        let!(:checklist_it1) { create :process_checklist, booking_type: appointment_type }
+        let!(:checklist_it2) { create :process_checklist, booking_type: appointment_type }
+        let!(:checklist_it3) { create :process_checklist, booking_type: appointment_type }
```

Comment:
Can you please check if we can make use of `create_list` here?

Reply:
@swatijadhav we cannot use `create_list` here, as the assigned variables are used